aten/src/ATen/native/quantized/cpu/conv_serialization.h

```diff
@@ -330,6 +331,37 @@ c10::intrusive_ptr<ConvPackedParamsBase<kSpatialDim>> deserialize_conv(
 
   auto& ctx = at::globalContext();
 
+#ifdef USE_FBGEMM
+  if (ctx.qEngine() == at::QEngine::X86) {
+#if AT_MKLDNN_ENABLED()
```

Comment:
Should this really be conditioned on `USE_FBGEMM`? I.e. if MKLDNN is enabled and USE_FBGEMM is not should this codepath be still enabled?

Reply:
Thanks for the question. The design is that the unified backend (x86) is enabled if and only if FBGEMM is available.
https://github.com/pytorch/pytorch/blob/53998a157e3bc7f08517bee0802492aa293a1a77/aten/src/ATen/Context.cpp#L315